scripts/install2

```diff
@@ -11,6 +11,10 @@ CJDNS_BUILD_CMD="sudo Seccomp_NO=1 NO_NEON=1 ./do"
 if [[ $RPI_REVISION == *"900092"* || $RPI_REVISION == *"900093"* ]]; then
     echo -e "\e[1;32mStarting installation on Raspberry Pi Zero...\e[0m"
     CJDNS_BUILD_CMD="sudo Seccomp_NO=1 NO_NEON=1 CFLAGS=\"-s -static -Wall\" ./do"
+elif [[ `cat /proc/cpuinfo  | grep Hardware | awk '{print $3}'` == 'Allwinner' ]]; then
+    echo -e "\e[1;32mStarting installation on Orange Pi ???... \e[1;31m(EXPERIMENTAL) \e[0m"
+    CJDNS_BUILD_CMD="sudo Seccomp_NO=1 CFLAGS=\"-s -static -Wall -mfpu=neon -mcpu=cortex-a7 -mtune=cortex-a7 -fomit-frame-pointer -marm\" ./do"
+    RPI_REVISION="OPI"
```

Comment:
I tried not to make it as intrusive by introducing a second variable, but i agreea that board_family is a better fix.

I was speaking with the ARMBAIN people about identifying boards and they noted that it is impossible to identify  a Orange PI live vs Orange PI One board that is why they have different images for each.

I'm open to suggestions on how to detect it, or does it need to be stronger at this point in time.